BBallStatsTrack/src/com/example/bballstatstrack/models/Player.java

```diff
@@ -0,0 +1,265 @@
+package com.example.bballstatstrack.models;
+
+public class Player
+{
+    private static final String B_BALL_STAT_TRACK = "BBallStatTrack";
+
+    private int mNumber;
+
+    private String mFullName;
+
+    private int m2ptFGMiss = 0;
+
+    private int m2ptFGMade = 0;
+
+    private int m3ptFGMiss = 0;
+
+    private int m3ptFGMade = 0;
+
+    private int mFTMiss = 0;
+
+    private int mFTMade = 0;
+
+    private int mOffRebound = 0;
+
+    private int mDefRebound = 0;
+
+    private int mAssist = 0;
+
+    private int mTurnover = 0;
+
+    private int mSteal = 0;
+
+    private int mBlock = 0;
+
+    private int mFoul = 0;
+
+    private int mPlayingTimeSec = 0;
+
+    public Player( int id, String fullName )
+    {
+        setNumber( id );
+        setFullName( fullName );
+    }
+
+    public Player( int number, String fullName, int miss1pt, int miss2pt, int miss3pt, int made1pt, int made2pt,
```

Comment:
Basically as stats all fields are required but not all parameters needed to be initialized or passed in the constructor, for example
```
new Player( 12, "Player One", 0, 0, 3, 0, 0, 1, 0, 1, 0, 0, 1, 0, 3, 130);
```
Many stats are basically just zero and therefore can have basic values of zero, using a builder pattern will essentially do this by making unset values to zero.
___Example__
![image](https://cloud.githubusercontent.com/assets/7711811/12316895/c2ffb3e4-bac6-11e5-8e49-81866d58e0cd.png)
You can see that fields/parameters which has the potential to have a 0 value is not included in the constructor but instead has an optional setter method and only fields/parameters that needs/required to have a value is asked on the parameter.